packages/address-book-controller/package.json

```diff
@@ -22,9 +22,10 @@
   ],
   "scripts": {
     "changelog:validate": "../../scripts/validate-changelog.sh @metamask/address-book-controller",
-    "doc": "../../scripts/doc.sh",
+    "doc": "typedoc",
     "test": "jest",
-    "test:watch": "jest --watch"
+    "test:watch": "jest --watch",
+    "doc:clean": "rimraf docs"
```

Comment:
I'm pretty sure TypeDoc does this by default as well. 😄

Reply:
Ah ok :) Removed in [a015279](https://github.com/MetaMask/controllers/pull/831/commits/a015279fed5bb52b1e06abd340fe42dfa3a6fc43).